### PR TITLE
OR-1108 add some more rules

### PR DESCRIPTION
**Pull request by nguyenzung**
I added some more rules on Makefile

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+run: $(TARGET)
+	$(TARGET)
+
+$(BUILD_DIR):
+	mkdir -p $@
+
+$(TARGET): build
 
 build:
 	CGO_ENABLED=0 GOOS=linux go build -o ./build/bin/titond ./cmd/titond/main.go
```
Comment (rlgns98kr) [inline Makefile:18]: Thank you very much Brave! 😍
Can you change `./build/bin/titond` to $TARGET ?

File: Makefile [hunk 2]
```diff
@@ -9,3 +23,13 @@ titond:
...
 	@echo "Done building"
 	@echo "Run \"./build/bin/titond\" to launch titond backend."
+
+image:
+	docker build --build-arg TARGETARCH=amd64 --build-arg TARGETOS=linux -t titond-backend .
+
+image-arm:
```
Comment (rlgns98kr) [inline Makefile:30]: I think we can add image-amd! And we can remove TARGETARCH in `image`. 🙇🏼‍♂️

File: Makefile (hunk 1 above)
Comment (nguyenzung) [reply Makefile:13]: Thank you Steven
I updated!

File: Makefile (hunk 2 above)
Comment (nguyenzung) [reply Makefile:28]: Yes, so image rule is not only for amd64
I updated!

**Review by rlgns98kr — APPROVED** [review]
Thank you very much, Brave 😍

**Pull request merged**